### PR TITLE
Add instructions to run python locally

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- adjust runtime to allow run function locally
- add readme how to run function locally

**Related issue(s)**

### CODE REVIEW

File: components/function-runtimes/python39/kubeless/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## How to locally run and debug Python Function nad runtime
```
Comment (grego952) [inline components/function-runtimes/python39/kubeless/README.md:1]:
```suggestion
## How to locally run and debug Python Function and runtime
```

File: components/function-runtimes/python39/kubeless/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+* Create [venv](https://docs.python.org/3/library/venv.html)
+* Active venv:
```
Comment (grego952) [inline components/function-runtimes/python39/kubeless/README.md:4]:
```suggestion
* Activate venv:
```

File: components/function-runtimes/python39/kubeless/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+## How to locally run and debug Python Function nad runtime
+
+* Create [venv](https://docs.python.org/3/library/venv.html)
```
Comment (grego952) [inline components/function-runtimes/python39/kubeless/README.md:3]: These steps must be done in this order, right?
If so, can you change the unordered list to the ordered one?

File: components/function-runtimes/python39/kubeless/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+* Active venv:
+  ```bash
+  source venv/bin/activete
+  ```
+* Create the `function` directory with `handler.py` and `requirements.txt`
+* Install dependencies from runtime and function:
```
Comment (grego952) [inline components/function-runtimes/python39/kubeless/README.md:9]:
```suggestion
* Install dependencies from runtime and Function:
```

File: components/function-runtimes/python39/kubeless/README.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    pip install -r requirements.txt
+    pip install -r kubeless/requirements.txt
+    ```
+* Set following envs:
+    ```bash
+    export FUNCTION_PATH=./function
+    export MOD_NAME=handler
+    export FUNC_HANDLER=main
+    ```
+* Run function from the terminal.
```
Comment (grego952) [inline components/function-runtimes/python39/kubeless/README.md:20]:
```suggestion
* Run Function from the terminal.
```

File: components/function-runtimes/python39/kubeless/README.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  ```bash
+  source venv/bin/activete
+  ```
+* Create the `function` directory with `handler.py` and `requirements.txt`
+* Install dependencies from runtime and function:
+    ```bash
+    pip install -r requirements.txt
+    pip install -r kubeless/requirements.txt
+    ```
+* Set following envs:
```
Comment (grego952) [inline components/function-runtimes/python39/kubeless/README.md:14]:
```suggestion
* Set the following envs:
```

**Review by grego952 — APPROVED** [review]

**Review by MichalKalke — APPROVED** [review]

**Pull request merged**